frontend/src/pages/Overview/OverviewPage.tsx

```diff
@@ -159,6 +159,7 @@ type ReduxProps = {
   refreshInterval: IntervalInMilliseconds;
   minTLS: string;
   istioAPIEnabled: boolean;
+  isMaistra: boolean;
```

Comment:
Take this opportunity to fix the alphabetization of these props.

Reply:
done